src/foam/comics/v2/DAOBrowseControllerView.js

```diff
@@ -41,12 +41,13 @@ foam.CLASS({
 
   css: `
     ^container {
-      padding: 24px 32px 16px 32px;
+      padding: 0;
```

Comment:
0 padding seems like a bad idea, maybe replace with 
```suggestion
      padding: '8px 16px 8px 16px';
```